app/experimenter/experiments/migrations/0164_auto_20210401_1758.py

```diff
@@ -0,0 +1,84 @@
+# Generated by Django 3.1.7 on 2021-04-01 17:58
+
+from django.db import migrations, models
+
+
+def update_statuses(apps, schema_editor):
+    NimbusExperiment = apps.get_model("experiments", "NimbusExperiment")
+    NimbusChangeLog = apps.get_model("experiments", "NimbusChangeLog")
+    db_alias = schema_editor.connection.alias
+    NimbusExperiment.objects.using(db_alias).filter(status="Review").update(
+        status="Draft", publish_status="Approved"
+    )
+    NimbusExperiment.objects.using(db_alias).filter(status="Accepted").update(
+        status="Draft", publish_status="Waiting"
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(old_status="Review").update(
+        old_status="Draft",
+        old_publish_status="Review",
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(old_status="Accepted").update(
+        old_status="Draft",
+        old_publish_status="Waiting",
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(new_status="Review").update(
+        new_status="Draft",
+        new_publish_status="Waiting",
```

Comment:
`"Approved"` here?

Reply:
@jaredlockhart if the status was Review isn't it currently in RS waiting for action?

Reply:
No `REVIEW` was the old version of `APPROVED` ie waiting to be PUSHED to remote settings, not yet in remote settings